tests/tests_integration/test_api/test_functions.py

```diff
@@ -31,6 +31,58 @@ def a_function(cognite_client: CogniteClient) -> Function:
     return function
 
 
+@pytest.fixture(scope="session")
+def dummy_function(cognite_client: CogniteClient) -> Function:
+    name = "integration_test-dummy-noxid-1"
+    retrieved = cognite_client.functions.list(name=name, limit=1)
+    if retrieved:
+        return retrieved[0]
+
+    def handle(client, data, secrets, function_call_info):
+        print(f"Inputs: {data!r}")  # noqa
+        print(f"Call info: {function_call_info!r}")  # noqa
+        return data
+
+    created = cognite_client.functions.create(
+        name=name, function_handle=handle, description="print inputs & call info, return inputs"
+    )
+    schedules = [
+        FunctionScheduleWrite(
+            name="once-a-year-05-13",
+            cron_expression="36 2 13 5 *",
+            function_id=created.id,
+            data={"day": "05-13", "scheduled": True},
+        ),
+        FunctionScheduleWrite(
+            name="once-a-year-04-03",
+            cron_expression="6 4 3 4 *",
+            function_id=created.id,
+            data={"day": "04-03", "scheduled": True},
+        ),
+        FunctionScheduleWrite(
+            name="once-a-year-06-06",
+            cron_expression="50 1 6 6 *",
+            function_id=created.id,
+            data={"day": "06-06", "scheduled": True},
+        ),
+        FunctionScheduleWrite(
+            name="once-a-year-06-24",
+            cron_expression="14 6 24 6 *",
+            function_id=created.id,
+            data={"day": "06-24", "scheduled": True},
+        ),
+        FunctionScheduleWrite(
+            name="once-a-year-05-25",
+            cron_expression="36 6 25 5 *",
+            function_id=created.id,
+            data={"day": "05-25", "scheduled": True},
+        ),
+    ]
```

Comment:
Why do we need 5 schedules? 😜 (Not that it really matters with once-a-year crons)

Reply:
Just matching exactly what was already there :) Did not want to start modifying anything unnecessarily